psalm-baseline.xml

```diff
@@ -1019,6 +963,11 @@
       <code><![CDATA[CallbackHandler]]></code>
     </UndefinedDocblockClass>
   </file>
+  <file src="src/Validator/Csrf.php">
```

Comment:
We really shouldn't be expanding the baseline while moving towards 3 - This is likely due to dropping the defined array shape on the CSRF validator